flink-autoscaler-plugin-jdbc/pom.xml

```diff
@@ -140,6 +140,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>5.1.0</version>
+        </dependency>
```

Comment:
1. The `HikariCP` is production dependency, it's better to move it before test related dependencies.
2. The version is better to extract a property.